desktop_photo_search/fluent_ui/pubspec.yaml

```diff
@@ -0,0 +1,55 @@
+name: desktop_photo_search
+description: Search for Photos, using the Unsplash API.
+publish_to: 'none'
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.15.0 <3.0.0"
+
+dependencies:
+  built_collection: ^5.1.1
+  built_value: ^8.1.4
+  cupertino_icons: ^1.0.2
+  file_selector: ^0.8.3
+  file_selector_linux: ^0.0.2+1
+  file_selector_macos: ^0.0.4+1
+  file_selector_windows: ^0.8.2
+  fluent_ui: ^3.7.0
+  fluentui_system_icons: ^1.1.157
+  flutter:
+    sdk: flutter
+  http: ^0.13.4
+  logging: ^1.0.2
+  menubar:
+    git:
+      url: https://github.com/google/flutter-desktop-embedding.git
+      path: plugins/menubar
```

Comment:
Is this something that'll get published in the future?

Reply:
There is ongoing work on design of the API for menus.